tests/column_aggregate_expectations/expect_column_median_to_be_between.json

```diff
@@ -1,128 +1,170 @@
-        "exact_match_out": false,
-        "in": {
-          "column": "c",
-          "min_value": 6,
-          "max_value": 6,
-          "catch_exceptions": true
+        {
+          "title": "Negative test, null min",
+          "exact_match_out": false,
+          "in": {
+            "column": "b",
+            "min_value": null,
+            "max_value": 1
+          },
+          "out": {
+            "success": false,
+            "observed_value": 2
+          }
         },
-        "out": {
-          "success": true,
-          "observed_value": 6
-        }
-      },{
-        "title": "Negative Test, missing value in column, COMPLETE result format",
-        "exact_match_out": true,
-        "in": {
-          "column": "c",
-          "min_value": 7,
-          "max_value": 7,
-          "result_format": "COMPLETE"
+        {
+          "title": "Negative test, null max",
+          "exact_match_out": false,
+          "in": {
+            "column": "b",
+            "min_value": 2.5,
+            "max_value": null
+          },
+          "out": {
+            "success": false,
+            "observed_value": 2
+          }
         },
-        "out": {
-          "success": false,
-          "result": {
-            "observed_value": 6.0,
-            "element_count": 4,
-            "missing_count": 1,
-            "missing_percent": 0.25
+        {
+          "title": "Positive Test, missing value in column, exact min and max",
+          "exact_match_out": false,
+          "in": {
+            "column": "c",
+            "min_value": 6,
+            "max_value": 6,
+            "catch_exceptions": true
+          },
+          "out": {
+            "success": true,
+            "observed_value": 6
           }
-      }
-      },{
-        "title": "Test exception, both min and max null",
+        },
+        {
+          "title": "Negative Test, missing value in column, COMPLETE result format",
+          "exact_match_out": true,
+          "in": {
+            "column": "c",
+            "min_value": 7,
+            "max_value": 7,
+            "result_format": "COMPLETE"
+          },
+          "out": {
+            "success": false,
+            "result": {
+              "observed_value": 6.0,
+              "element_count": 4,
+              "missing_count": 1,
+              "missing_percent": 0.25
+            }
+          }
+        },
+        {
+          "title": "Test exception, both min and max null",
+          "exact_match_out": false,
+          "in": {
+            "column": "c",
+            "min_value": null,
+            "max_value": null,
+            "catch_exceptions": true
+          },
+          "out": {},
+          "error": {
+            "traceback_substring": "min_value and max_value cannot both be None"
+          }
+        }
+      ]},
+    {
+      "data": {
+        "empty_column": []
+      },
+      "tests": [{
+        "title": "Test empty column",
```

Comment:
"Test empty column" is a bit generic of a test description, maybe you could provide some info about the expected behavior?